doc/tfm/conf.py

```diff
@@ -25,7 +25,7 @@
 project = "Trusted Firmware-M"
 copyright = "2017-2021, ARM CE-OSS"
 author = "ARM CE-OSS"
-version = "1.8.0"
+version = "2.0.0"
```

Comment:
Updated as 2.0.0 as confirmed by @shanthanordic . 
@nvlsianpu : Could you confirm the version updates for MCUboot.

Reply:
base version of mcuboot upstream is post 2.0.0